src/core/Icon/map.ts

```diff
@@ -77,11 +79,13 @@ import { ReactComponent as IconTreeHorizontalPrivateLarge } from "../../common/s
 import { ReactComponent as IconTreeHorizontalPublicLarge } from "../../common/svgs/IconTreeHorizontalPublicLarge.svg";
 import { ReactComponent as IconTreeHorizontalSmall } from "../../common/svgs/IconTreeHorizontalSmall.svg";
 import { ReactComponent as IconTreeVerticalSmall } from "../../common/svgs/IconTreeVerticalSmall.svg";
+import { ReactComponent as IconVirusLarge } from "../../common/svgs/IconVirusLarge.svg";
 import { ReactComponent as IconXMarkCircleSmall } from "../../common/svgs/IconXMarkCircleSmall.svg";
 import { ReactComponent as IconXMarkLarge } from "../../common/svgs/IconXMarkLarge.svg";
 import { ReactComponent as IconXMarkSmall } from "../../common/svgs/IconXMarkSmall.svg";
 
 export interface IconNameToSizes {
+  bacteria: "l";
```

Comment:
typically we allow `l` and `xl` for an icon with a large image. Any reason why that isn't the case here?

Reply:
Since the image is titled "Large," I thought there should only be a large size. I'll now add the xl sizes. 🥸